react/features/display-name/components/web/StageParticipantNameLabel.tsx

```diff
@@ -43,15 +48,15 @@ const useStyles = makeStyles(theme => {
  */
 const StageParticipantNameLabel = () => {
     const classes = useStyles();
-    const largeVideoParticipant = useSelector(getLargeVideoParticipant);
+    const largeVideoParticipant: Participant = useSelector(getLargeVideoParticipant);
     const selectedId = largeVideoParticipant?.id;
-    const nameToDisplay = useSelector(state => getParticipantDisplayName(state, selectedId));
+    const nameToDisplay = useSelector((state: IState) => getParticipantDisplayName(state, selectedId));
 
     const localParticipant = useSelector(getLocalParticipant);
     const localId = localParticipant?.id;
 
     const isTileView = useSelector(isLayoutTileView);
-    const toolboxVisible = useSelector(isToolboxVisible);
+    const toolboxVisible: boolean = useSelector(isToolboxVisible);
```

Comment:
any reason why only some const assignments mention their type?

Reply:
Some variables need explicit types because the selector function is not yet in TS so we don't know it's return type